rpa-manager/modules/ROOT/pages/settings-adminsettings-reassignassets.adoc

```diff
@@ -0,0 +1,86 @@
+= Reassigning RPA Assets
+
+If an RPA user or team is deleted or disabled in Anypoint Access Management, RPA assets that were assigned to this user or team are shown in the *Reassign Assets* tab in the *Admin Settings* view of the *Settings* module.
+
+All users with the *Reassign Users* or *RPA Administrator* permission get an email when an RPA asset is orphaned because its last assigned user is deleted.
+
+You can reassign the following RPA assets to another user:
+
+* Credentials
+* Process Evaluations 
+* Process Projects
+* Process Board Tasks 
+* Alert Targets
+
+The following assets are deleted on the deletion of the user:
+
+* Private dashboards
+* User API Keys
+
+The following assets are not shown in the *Reassign Assets* view and can be reassigned via other views:
+
+* User Tasks via the *Unprocessed User Tasks* view
+
+== Before You Begin
+
+* Ask an Anypoint Access Management administrator to assign you the required permissions:
++
+[cols="1,2,2"]
+|===
+|*Action* |*RPA Permission (deprecated)* | *RPA Permission*
```

Comment:
Suggestion: RPA Permission > Deprecated RPA Permission
This distinguishes the column head more from the RPA Permission column head.